man/pam_systemd_loadkey.xml

```diff
@@ -49,9 +49,44 @@
         <term><varname>keyname=</varname></term>
 
         <listitem><para>Takes a string argument which sets the keyname to read.
-        The default is <literal>cryptsetup</literal>, which is used by
-        <citerefentry><refentrytitle>systemd-cryptsetup@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
-        to store LUKS passphrase during boot.</para>
+        The default is <literal>cryptsetup</literal>.
+        During boot, <citerefentry><refentrytitle>systemd-cryptsetup@.service</refentrytitle><manvolnum>8</manvolnum></citerefentry> stores a passphrase or PIN in the keyring.
+        The LUKS2 volume key can also be used, via the <option>link-volume-key</option> option in <citerefentry><refentrytitle>crypttab</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
```

Comment:
Could you break the long lines? We typically breaks line with 109ch, though, not strict, and sometimes longer e.g. the line that has e.g. `<citerefentry>` or so.

Reply:
Sure, should be fixed now. Output looks identical to me. 

The `<citerefentry>` line is still too long but I think that's OK, if I understood you correctly?